frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -438,6 +440,12 @@ export const insightLogic = kea<insightLogicType>({
                 setTagLoading: (_, { tagLoading }) => tagLoading,
             },
         ],
+        sourceDashboardId: [
+            null as number | null,
+            {
+                setSourceDashboardId: (_, { dashboardId }) => dashboardId,
+            },
+        ],
```

Comment:
Conceptually this and actually the entire `saveInsight` feels like they make more sense on `insightSceneLogic`... 🤔